shared/naturalcrit/markdown.js

```diff
@@ -304,10 +306,19 @@ const definitionListsSingleLine = {
 		let endIndex = 0;
 		const definitions = [];
 		while (match = regex.exec(src)) {
```

Comment:
I recognize it is a little excessive to run this regex here where the `match` result isn't used anymore (except to grab the full line). But I'm hoping to leave this extension mostly intact until V4 comes around where we can do a proper cleanup.

Reply:
I think keeping it clean now will help us later on, we might not remember to do this or we might have other PRs in mind. If it's not too much to ask.